README.md

```diff
@@ -170,6 +170,59 @@ uart:
     direction: BOTH
 ```
 
+## Preserving your batteries
```

Comment:
Could you move this to the `Goodies` section and use a sub-headline? (`### Preserving your batteries`)